src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivitiesService.kt

```diff
@@ -218,7 +266,9 @@ class ActivitiesService(
       when {
         existingPayRate == null -> newPayRates.add(requestedPayRate.toCourseActivityPayRate(existingActivity))
         existingPayRate.rateIsUnchanged(requestedPayRate) -> newPayRates.add(existingPayRate)
-        existingPayRate.rateIsChangedButNotYetActive(requestedPayRate) -> newPayRates.add(existingPayRate.apply { halfDayRate = requestedPayRate.rate }) // e.g. rate adjusted twice in same day
+        existingPayRate.rateIsChangedButNotYetActive(requestedPayRate) -> newPayRates.add(
+          existingPayRate.apply { halfDayRate = requestedPayRate.rate }
+        ) // e.g. rate adjusted twice in same day
```

Comment:
had to reformat as ktlint didnt like existing code 🤷

Reply:
That's a bit weird that it didn't break ktlint on the last build (or locally for me). Are we using the same linter? It's not a big deal but the comment beneath the code makes it look like it applies to the next line - so if we need this formatting maybe the comment is better on the previous line?

Reply:
ill move the comment